websites/T/Territorial/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+		clientId: "1014903980410802237",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/8vXevly.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
```

Comment:
`Assets.Logo` is only used once, you may as well just specify it here
```suggestion
			largeImageKey: "https://i.imgur.com/8vXevly.png",
```

Reply:
okay

Reply:
not sure if this helps, but this is the favicon for the website: https://territorial.io/favicon.ico